Pod/Support/OSX/RunTestsTargetWithSlimPort

```diff
@@ -2,19 +2,72 @@
 source $1
 ENVIRONMENT_SRC=$1
 SLIM_PORT=$2
-APPLICATION_PATH=${TARGET_BUILD_DIR}/${EXECUTABLE_PATH}
+EXECUTABLE_PATH=${TARGET_BUILD_DIR}/${EXECUTABLE_PATH}
+APP_PATH=${TARGET_BUILD_DIR}/${PRODUCT_NAME}.app
 
-run_test_system() {
-	echo "Test System Executable Folder Path: $APPLICATION_PATH"
-	$APPLICATION_PATH $SLIM_PORT
-	return $?
-}
+# Main Subroutines
 
 main() {
-	echo "== OCSlimProject: RunSlimTestTargetWithSlimPort~OSX:$SLIM_PORT =="
-	echo "Test System Executable Environment: $ENVIRONMENT_SRC"
+	log "== RunSlimTestTargetWithSlimPort~macOSX:$SLIM_PORT =="
 	run_test_system
 	exit $?
 }
 
-main
\ No newline at end of file
+run_test_system() {
+	log "Test System Executable = $EXECUTABLE_PATH"
+	local LOGFILE=$(get_log_file_path)
+	launch_osx_app $LOGFILE
+	write_logs_to_console $LOGFILE
+	log "*** App Exited ***"
+	return $?
+}
+
+launch_osx_app() {
+	log "*** Launching ${PRODUCT_NAME}.app ***"
+	local LOGFILE=$1
+	$EXECUTABLE_PATH $SLIM_PORT > $LOGFILE 2>&1
+}
+
+## Logging Subroutines
+
+write_logs_to_console() {
+	local LOGFILE=$1
+	tidy_zero_line_app_log_file $LOGFILE
+	if [ $? -eq 1 ]; then
+		cat $LOGFILE 
+	fi
+}
+
+get_log_file_path() {
+	local LOGDIR="Logs"
+	local LOGPATH="$(pwd)/$LOGDIR"
+	local LOGFILENAME="$(date +%Y%m%d-%H%M%S).log"
+	local LOGFILE="$LOGPATH/$LOGFILENAME"
+	mkdir -p $LOGPATH
+	echo $LOGFILE
+}
+
+tidy_zero_line_app_log_file(){
```

Comment:
I don't this is necessary.
There's implicit information in a log file that is empty.
Additionally, the log should always contain something (e.g. `start_slim()` would write something),  no?

Reply:
Logs now always generate some head and tail output. Files not deleted 👍